go/packages/packages_linux.go

```diff
@@ -116,15 +164,17 @@ func aptUpgrade() error {
 	return nil
 }
 
+// update yum packages
 func yumUpdate() error {
-	if _, err := exec.Command(yum, yumCheckUpdatesArgs...).CombinedOutput(); err != nil {
+	if _, err := exec.Command(yum, yumUpdateArgs...).CombinedOutput(); err != nil {
```

Comment:
Wrap this in run as well? Looks like this was missed for all the yum commands previously.

Reply:
tthe 'run' is just a wrapper around .CombinedOutput method with some extra logging. I believe it was created for code re-usability.
Over here, i think, we forgot to use run and instead called the combinedoutput directly. i will change it to use run since that makes it consistent with rest of the code and also helps in writing test for this method.